src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -1851,7 +1851,8 @@ private void checkCipherState() {
      * new block
      *
      * @throws IllegalStateException if this {@code Cipher} object is in a
-     * wrong state (e.g., has not been initialized)
+     * wrong state (e.g., has not been initialized, or is not
+     * in ENCRYPT_MODE or DECRYPT_MODE)
```

Comment:
Use code format when referencing the modes, ex: change this line to:

`* in {@code ENCRYPT_MODE} or {@code DECRYPT_MODE})`

Same comment applies to all other applicable lines below.